pb.h

```diff
@@ -518,10 +518,25 @@ struct pb_extension_s {
     pb_membersize(st, u.m[0]), 0, ptr}
 
 #define PB_ONEOF_FIELD(union_name, tag, type, rules, allocation, placement, message, field, prevfield, ptr) \
-        PB_ ## rules ## _ ## allocation(union_name, tag, message, field, \
```

Comment:
I also snuck in this change and forgot to revert it, but I think it is safely explicit in this case